packages/keymanager-server/src/impl.ts

```diff
@@ -0,0 +1,226 @@
+import {SecretKey} from "@chainsafe/bls";
+import {Keystore} from "@chainsafe/bls-keystore";
+import {
+  Api,
+  DeletionStatus,
+  ImportStatus,
+  KeystoreStr,
+  SlashingProtectionData,
+} from "@chainsafe/lodestar-api/keymanager";
+import {fromHexString} from "@chainsafe/ssz";
+import {Interchange, ISlashingProtection, SecretKeyInfo, SignerType, Validator} from "@chainsafe/lodestar-validator";
+import {PubkeyHex} from "@chainsafe/lodestar-validator/src/types";
+import {Root} from "@chainsafe/lodestar-types";
+import {unlink, writeFile} from "fs/promises";
+import lockfile from "lockfile";
+import {ILogger} from "@chainsafe/lodestar-utils";
+
+export const LOCK_FILE_EXT = ".lock";
+
+export class KeymanagerApi implements Api {
+  constructor(
+    private readonly logger: ILogger,
+    private readonly validator: Validator,
+    private readonly slashingProtection: ISlashingProtection,
+    private readonly genesisValidatorRoot: Uint8Array | Root,
```

Comment:
I think these two should be removed here and the functionality exported from `Validator`.
```ts
class Validator {
...
  async importInterchange(interchange: Interchange): Promise<void>
  async exportInterchange(pubkeys: PubkeyHex[], version: InterchangeFormatVersion = {version: "5"}): Promise<Interchange>
```

Reply:
Implemented suggestion